src/Tizen.Sensor/Tizen.Sensor/Plugins/HeartRateMonitorBatch.cs

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Tizen.Sensor.BatchData;
+
+namespace Tizen.Sensor
+{
+    /// <summary>
+    /// The HeartRateMonitorBatch class is used for TODO.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public sealed class HeartRateMonitorBatch : BatchSensor<HeartRateMonitorBatchData>
+    {
+        private static string HRMBatchKey = "http://tizen.org/feature/sensor.heart_rate_monitor.batch";
+
+        private event EventHandler<SensorAccuracyChangedEventArgs> _accuracyChanged;
+
+        protected override IReadOnlyList<HeartRateMonitorBatchData> ConvertBatchData()
+        {
+            List<HeartRateMonitorBatchData> list = new List<HeartRateMonitorBatchData>();
+            Interop.SensorEventStruct sensorData;
+            for (int i = 0; i < BatchedEvents.Count; i++)
+            {
+                sensorData = BatchedEvents[i];
+                list.Add(new HeartRateMonitorBatchData(new TimeSpan((Int64)sensorData.timestamp), sensorData.accuracy, (HeartRateMonitorBatchState)sensorData.values[0], (int)sensorData.values[1], (int)sensorData.values[2]));
```

Comment:
What is timestamp from native?

TimeSapn constructor use tick
https://docs.microsoft.com/en-us/dotnet/api/system.timespan.-ctor?view=netframework-4.8#System_TimeSpan__ctor_System_Int64_

Reply:
@myroot It's tick.
Here's how to get that value.
```
unsigned long long util::get_timestamp(clockid_t clk_id)
{
	struct timespec t;
	clock_gettime(clk_id, &t);
	return (unsigned long long)(SEC_TO_USEC(t.tv_sec) + NSEC_TO_USEC(t.tv_nsec));
}
```